package.json

```diff
@@ -20,6 +20,7 @@
   },
   "scripts": {
     "start": "cross-env NODE_ENV=development SITE_ID=local SITE_HOST=localhost npm run start:common",
+    "start:mini": "cross-env NODE_ENV=development SITE_ID=local SITE_HOST=localhost npm-run-all --parallel server watch:extension",
```

Comment:
Don't absolutely need this, but it came in handy for saving some CPU on my machine when I just wanted to build the site and not also lint & test on every change.